### ISSUE TITLE
amp-recaptcha-input: Document how AMP handles subdomains (Especially in the case of www)

### ISSUE THREAD

**Issue by torch2424** [ampproject/amphtml#22478]
relates to #22279

We should be clear that amp wont make any assumptions with URL domains, so both:

`www.example.com` and `example.com`

This should be clear and documented. 😄
